app/main.js

```diff
@@ -28,7 +28,7 @@ const router = new VueRouter({
     {
       path: '/login',
       component: Login,
-      meta: { hidenActionBar: true }
+      meta: { hiddenActionBar: true }
```

Comment:
this meta info appears not to be being used but it was misspelled so just fixed that.

Reply:
Yup. I was using that to show/hide the action bar. I will delete it soon.